fetch/nosniff/parsing-nosniff.html

```diff
@@ -3,7 +3,7 @@
 <div id=log></div>
 <script>
```

Comment:
Mind adding a comment, while you're here, on just what this test is doing?  It looks to me like an accurate description would be something like:

Tests whether variations of a "X-Content-Type-Options: nosniff" header can be parsed as such. When the response header fails to be parsed as a valid nosniff header, the advertised Content-Type is not recognized as a possible script, and the script's onerror handler is invoked.  When the nosniff header is not parsed as a valid nosniff header, the MIME type of the response is sniffed as text, and the onload handler is run.